includes/templates/responsive_classic/css/stylesheet.css

```diff
@@ -259,14 +249,14 @@ TR.tableHeading {}
 #checkoutOrderTotals div, #csNotifications .buttonRow{margin-right:20px;}
 #checkoutPayment label, #csNotifications input{margin-left:20px;}
 div#checkoutShippingContentChoose{margin-bottom:20px;}
-#sendSpendWrapper h2{margin-top:-20px;}
+#sendSpendWrapper h2{padding:10px;font-weight:normal;margin-top:-20px;margin-bottom:0;text-align:center;}
 
 /*bof images*/
-#productMainImage, #reviewsInfoDefaultProductImage, #productReviewsDefaultProductImage, #reviewWriteMainImage {margin:0 1em 1em 0;}
+#reviewsInfoDefaultProductImage, #productReviewsDefaultProductImage {margin:0 1em 1em 0;}
```

Comment:
Your comments say:
```
#productMainImage{float:none;margin:0;}
#productMainImage{margin:0 1em 1em 0;}

#reviewWriteMainImage{float:none;margin:0;}
#reviewWriteMainImage{margin:0 1em 1em 0;}
```

... but I'm only seeing the removal here, not the relocation or combination.

Reply:
Found it. Existing content, not part of diff "changes".